pkgs/development/compilers/polyml/default.nix

```diff
@@ -10,6 +10,13 @@ stdenv.mkDerivation rec {
   pname = "polyml";
   version = "5.9.1";
 
+  src = fetchFromGitHub {
+    owner = "polyml";
+    repo = "polyml";
+    rev = "v${version}";
+    sha256 = "sha256-72wm8dt+Id59A5058mVE5P9TkXW5/LZRthZoxUustVA=";
+  };
+
```

Comment:
To keep the diff minimal, I suggest this either gets moved back to where it was, or at least put this in a separate commit stating you've moved it.

Reply:
Ok, I will split it into two commits.